lib/spack/spack/cmd/checksum.py

```diff
@@ -134,7 +134,9 @@ def checksum(parser, args):
     for line in version_lines.splitlines():
         v_match = d_splitter.match(line)
         url = vstring_url_dict[v_match.group(3)]
-        if url in pkg.version_urls().values() or url != pkg.url_for_version(ver(v_match.group(3))):
```

Comment:
Why do you think this is a typo?
https://github.com/FNALssi/spack/blob/27b3e05edf9ef03dc5a1b031fe988a735e7afb08/lib/spack/spack/version.py#L1129-L1142

Reply:
Because it is returning a not-defined error... Perhaps we should instead add it to the import list, which currently imports Version?